libs/filamentjs/tests/test.py

```diff
@@ -1,47 +1,39 @@
 #!/usr/bin/env python3
 
 import glob
-import http.server
 import os
 import shutil
-import socketserver
 import sys
 
 from pathlib import Path
 
+from twisted.web.server import Site, GzipEncoderFactory
```

Comment:
Maybe we should commit a `requirements.txt` file to make installing dependencies like this easier.

Reply:
Good idea, alternatively we could commit a Pipenv.  The test server is more for personal use right now, for Kokoro-based testing we'd need something more sophisticated.